src/commcare_cloud/commands/ansible/downtime.py

```diff
@@ -81,19 +82,29 @@ def wait_for_all_processes_to_stop(environment, ansible_context):
         if not still_running:
             break
 
-        options = ['abort', 'wait', 'kill']
-        response = ask_option(
-            "Some processes are still running.\n"
-            "Do you with do abort, wait or kill running processes?\n   ",
+        options = ['abort', 'wait', 'continue', 'kill',]
+        response = ask_option(inspect.cleandoc(
+            """Some processes are still running. Do you want to:"
+             - abort downtime"
+             - wait for processes to stop"
+             - continue with downtime regardless"
+             - kill running processes   
+            """),
             options,
-            options + ['a', 'w', 'k']
+            options + ['a', 'w', 'c', 'k']
         )
         if response in ('a', 'abort'):
-            downtime = get_downtime_record(environment)
-            cancel_downtime_record(environment, downtime)
-            return
+            if ask('This will start all CommCare processes again. Do you want to proceed?'):
+                downtime = get_downtime_record(environment)
+                supervisor_services(environment, ansible_context, 'start')
```

Comment:
@snopoke should this be a restart instead of start? thinking of this as a state where some processes are still running and some have stopped. Using start would mean that some of them would fail because they are already running but restart would safely just restart them.

Reply:
By this point the processes are already 'stopped' according to supervisor. Any remaining processes are either zombies or not controlled by supervisor so I don't expect any to be in the 'RUNNING' state. Also when you do `start all` supervisor will ignore any processes that are already running.